modules/doc/content/help/inl/applications.md

```diff
@@ -10,10 +10,79 @@ breadth of applications continues to expand as well. The following sections high
+
+!col small=12 medium=4 large=4
+!media application_logos/Griffin.png style=width:100%;
+!row-end!
+
+## Grizzly
+
+!row!
+!col small=12 medium=8 large=8
+[Grizzly Website](https://grizzly-docs.hpcondemand.inl.gov/latest/)\\
+Grizzly models the degradation due to normal operating conditions of nuclear power plant systems, structures, and components. The code also simulates the ability of degraded components to safely perform under a variety of conditions. Grizzly can be applied to a variety of components. However, its development focused initially on the embrittlement of reactor pressure vessels and concrete structures. Vessels can degrade and facture due to irradiation and high temperatures, while concrete can degrade due to expansive alkali-silica reactions. Grizzly has capability to model the performance effect of these and other mechanisms.
+
+!col small=12 medium=4 large=4
+!media application_logos/Grizzly.png style=width:100%;
+!row-end!
+
+## MARMOT
+
+!row!
+!col small=12 medium=8 large=8
+[MARMOT Website](https://marmot-docs.hpcondemand.inl.gov/latest/)\\
+MARMOT is a mesoscale fuel performance code. As such, it can predict the evolution of the microstructure and material properties of fuels and claddings due to stress, temperature, and irradiation damage. MARMOT can, therefore, supply microstructure-based materials models to other code that works on engineering scale, which is larger than mesoscale, with an example being BISON. MARMOT solves equations involving solid mechanics and heat conduction using the finite element method.
+
+!col small=12 medium=4 large=4
+!media application_logos/Marmot.png style=width:100%;
+!row-end!
+
+## Pronghorn
+
+!row!
+!col small=12 medium=8 large=8
+[Pronghorn Website](https://pronghorn-docs.hpcondemand.inl.gov/latest/)\\
+Pronghorn is a multi-dimensional, coarse-mesh, thermal-hydraulics code for advanced reactors and is particularly well-suited to model gas-cooled pebble bed and prismatic reactors. It serves the intermediate fidelity realm situated between detailed computational fluid dynamics analysis and lumped system models.
+
+!col small=12 medium=4 large=4
+!row-end!
+
+## Relap-7
+
+!row!
+!col small=12 medium=8 large=8
+[Relap-7 Website](https://relap7-docs.hpcondemand.inl.gov/latest/)\\
+A next generation nuclear systems safety code, RELAP-7 takes advantage of advances in computer architecture, software design, numerical methods, and physical models for use in the Risk Informed Safety Margin Characterization (RISMC) methodology and in nuclear power plant (NPP) safety analysis. RELAP-7 is a more capable than its predecessors in the RELAP family due to better flow models, improved numerical approximations, the ability to handle long duration events like full life cycle fuel evaluations, and easy coupling to other simulation code.
```

Comment:
Replace the whole description. I suggest:

RELAP-7 is a two-phase thermal systems code based on MOOSE's thermal hydraulics module. RELAP-7 provides two-phase components suitable for LWRs provides closures for two-phase water from TRACE.